draft-ietf-quic-transport.md

```diff
@@ -1721,7 +1721,12 @@ it can associate the peer's response with the corresponding PATH_CHALLENGE.
 ## Path Validation Responses
 
 On receiving a PATH_CHALLENGE frame, an endpoint MUST respond immediately by
-echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame.
+echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame,
+unless it has PATH_RESPONSE frames buffered for the same destination connection
+ID and wishes to limit memory consumption.
+
+An endpoint MUST NOT store or retransmit a PATH_RESPONSE frame, as a new
```

Comment:
I prefer @kazuho's rewording here: "And endpoint MUST NOT send more than one PATH_RESPONSE in response to a PATH_CHALLENGE frame, as a new ..."

Reply:
Agreed, I like that as well. Will change!

Reply:
Changed (updated, and thanks @martinthomson!)